src/assessment.js

```diff
@@ -8,30 +8,31 @@
 //
 // NOTE: Line numbers are annotated line numbers, not editor line numbers.
 
-/*1*/ var alderaan = "Boom";
-/*2*/ var deathStar = function deathStar(){
-/*3*/   var laserQuantity = 1;
-/*4*/   var laserSize = "huge";
-/*5*/   var fire = function fire(){
-/*6*/     // some code
-/*7*/   };
+/*1*/ var alderaan = { name: 'Alderaan' };
+/*2*/ var deathStar = {
+/*3*/   laserQuantity: 1,
+/*4*/   laserSize: "huge",
+/*5*/   fire: function(){
+/*6*/     console.log(this);
+/*7*/   }
 /*8*/ };
 
+
 // Suppose we had declared another function between line 7 and 8. Would that
 // new function have access to the value stored in the variable `laseQuantity`
 // on line 3?
 //
 //     `true` or `false`
 //
-var q1 = /* your answer here */;
+var q1 = false;
```

Comment:
The function would be declared in the same scope as `fire`; it will have access to the local variables.